src/app/app.component.html

```diff
@@ -1,3 +1,4 @@
+<evt-bibliography></evt-bibliography>
```

Comment:
Should this component be visualised here?

Reply:
For now, yes.